app/lib/hesa/parsers/itt_record.rb

```diff
@@ -48,6 +48,18 @@ class IttRecord
         disability7: "F_DISABLE7",
         disability8: "F_DISABLE8",
         disability9: "F_DISABLE9",
+        itt_key: "F_ITTKEY",
+        rec_id: "F_RECID",
```

Comment:
@jhenderson The reason I kept this as `rec_id` was because we already have a collection_reference field which we set on the table. We set that ourselves depending on `Settings.current_collection_reference` in the SyncStudents service. As this ticket was about taking in the additional information, I haven't changed the existing `collection_reference` field for now - I thought it might be useful to retain that as it is for now. But we could name it something else?

On `itt_key` I think @claire-hughez was going to follow up on this and ask HESA what this is.